core/src/test/java/io/seata/core/rpc/RpcContextTest.java

```diff
@@ -0,0 +1,195 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.core.rpc;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.HashSet;
+
+/**
+ * RpcContext Test
+ * 
+ * @author kaitithoma
+ * @author Danaykap
```

Comment:
Is this test case completed by two people?

Reply:
Yes, me and my partner kaitithoma did this together a month ago.

Reply:
Good.thanks.